project/src/main/java/com/google/sps/data/Place.java

```diff
@@ -52,44 +52,95 @@
   public abstract String phone();
 
   /**
-   * @return The rating of the place, represented by a number between 1-5.
+   * @return the rating of the place, represented by a number between 1-5.
    */
-  public abstract int rating();
+  public abstract double rating();
 
   /**
    * @return the place’s price level, represented by a number between 0-4.
    */
   public abstract int priceLevel();
 
   /**
-   * @return Coordinate of the physical place.
+   * @return coordinate of the physical place.
```

Comment:
As we discussed, I changed (in the PlacesFetcher PR) the longitude and latitude fields to be one "location" field of type LatLng since it's what the Places and Maps API recieve and return. 
Anyway I don't care to change it again once this is merged, just wanted to remind.

Reply:
cool, I think it's best that you will change it in your PR, cause I for example don't have the required imports and all that :)